drivers/sensor/amg88xx/Kconfig

```diff
@@ -0,0 +1,105 @@
+# Kconfig - AMG88XX infrared thermopile sensor configuration options
+
+#
+# Copyright (c) 2017 Phytec Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig AMG88XX
+	bool
+	prompt "AMG88XX Infrared Thermopile Sensor"
+	depends on I2C
+	default n
+	help
+	  Enable driver for AMG88XX infrared thermopile sensor.
+
+config AMG88XX_NAME
+	string
+	prompt "Driver name"
+	default "AMG88XX"
+	depends on AMG88XX
+	help
+	  Device name identifying the AMG88XX sensor.
+
+config AMG88XX_I2C_ADDR
```

Comment:
You can also add a range attribute to the config to force it to be either 0x68 or 0x69.